docs/source/en/tutorials/fast_diffusion.md

```diff
@@ -52,87 +54,83 @@ pipe = StableDiffusionXLPipeline.from_pretrained(
+    <img src="https://huggingface.co/datasets/sayakpaul/sample-datasets/resolve/main/progressive-acceleration-sdxl/SDXL%2C_Batch_Size%3A_1%2C_Steps%3A_30_1.png" width=500>
 </div>
 
-_(We later ran the experiments in float16 and found out that the recent versions of torchao do not incur numerical problems from float16.)_
+<Tip>
 
-**Why bfloat16?** 
+In our later experiments with float16, recent versions of torchao do not incur numerical problems from float16.
 
-* Using a reduced numerical precision (such as float16, bfloat16) to run inference doesn’t affect the generation quality but significantly improves latency. 
-* The benefits of using the bfloat16 numerical precision as compared to float16 are hardware-dependent. Modern generations of GPUs tend to favor bfloat16. 
-* Furthermore, in our experiments, we bfloat16 to be much more resilient when used with quantization in comparison to float16.  
+</Tip>
 
-We have a [dedicated guide](../optimization/fp16) for running inference in a reduced precision. 
+Take a look at the [Speed up inference](../optimization/fp16) guide to learn more about running inference with reduced precision.
 
-## Running attention efficiently
+## Scaled dot product attention
 
-Attention blocks are intensive to run. But with PyTorch's [`scaled_dot_product_attention`](../optimization/torch2.0), we can run them efficiently. 
+Attention blocks are intensive to run. But with PyTorch's [scaled dot product attention](../optimization/torch2.0#scaled-dot-product-attention), it is a lot more efficient.
 
 ```python
 from diffusers import StableDiffusionXLPipeline
 import torch
 
 pipe = StableDiffusionXLPipeline.from_pretrained(
-	"stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.bfloat16
+    "stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.bfloat16
 ).to("cuda")
 
 prompt = "Astronaut in a jungle, cold color palette, muted colors, detailed, 8k"
 image = pipe(prompt, num_inference_steps=30).images[0]
 ```
 
-`scaled_dot_product_attention` improves the latency from 4.63 seconds to 3.31 seconds.
-
-<div align="center">
-
-<img src="https://huggingface.co/datasets/sayakpaul/sample-datasets/resolve/main/progressive-acceleration-sdxl/SDXL%2C_Batch_Size%3A_1%2C_Steps%3A_30_2.png" width=500>
+Scaled dot product attention improves the latency from 4.63 seconds to 3.31 seconds.
 
+<div class="flex justify-center">
+    <img src="https://huggingface.co/datasets/sayakpaul/sample-datasets/resolve/main/progressive-acceleration-sdxl/SDXL%2C_Batch_Size%3A_1%2C_Steps%3A_30_2.png" width=500>
 </div>
 
-## Use faster kernels with torch.compile
+## torch.compile
 
-Compile the UNet and the VAE to benefit from the faster kernels. First, configure a few compiler flags:
+PyTorch 2 includes `torch.compile` which uses fast and optimized kernels for the UNet and VAE. First, configure a few compiler flags (refer to the [full list](https://github.com/pytorch/pytorch/blob/main/torch/_inductor/config.py) for more options):
```

Comment:
It reads like `torch.compile()` only uses faster and optimized kernels for the UNet and VAE. Could we reword it a bit to clear that confusion?